lib/cocoapods/installer/xcode/pods_project_generator/pod_target_integrator.rb

```diff
@@ -33,7 +33,6 @@ def integrate!
             UI.section(integration_message) do
               target_installation_result.non_library_specs_by_native_target.each do |native_target, spec|
                 add_embed_frameworks_script_phase(native_target, spec)
-                add_copy_resources_script_phase(native_target, spec)
```

Comment:
what about when using `s.resource_bundle` does it copy them over?

Reply:
Nope. I'll work on an update to do the change only for s.resource's. Good catch!